### ISSUE TITLE
Missing "APS_mode: False" in gradnorm/mls/vim.yml?

### ISSUE THREAD

**Issue by zjysteven** [Jingkang50/OpenOOD#136]
Hi,

When evaluating gradnorm/mls/vim, there will be a `KeyError: 'APS_mode'` when executing `ood_evaluator.py`. It seems that this is because in gradnorm/mls/vim.yml the `APS_mode` field is simply missing.

**Comment by WillCheung2016** [comment]
It seems this argument is missing in many yml files in the postprocessors folder.

**Comment by Jingkang50** [comment]
Yes, I am fixing them all currently. Sorry for the inconvenient.. We can simply add `APS_mode = False` in the config.
